tests/load/locustfile.py

```diff
@@ -45,7 +65,22 @@ class ConnectionTaskSet(TaskSet):
 
     @task
     def test_basic(self):
+        """ Perform a "basic" transaction test.
+
+        Desktop Autopush clients use a websocket connection to exchange
+        JSON command and response messages. (See
+        [Autopush HTTP Endpoints for Notifications](https://mozilla-services.github.io/autopush-rs/http.html#push-service-http-api)
+        for details).
```

Comment:
Will this render in the file? This looks like a markdown style link.

Reply:
I think it depends on whatever you're using to render the docs. I believe that Sphinx has a [markdown](https://www.sphinx-doc.org/en/master/usage/markdown.html) plug in. 

I tend to use markdown because I find it easier to read than RST.